tests/vectordb/test_elasticsearch_db.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+from unittest.mock import MagicMock, Mock, patch
+
+from embedchain.vectordb.elasticsearch_db import EsDB
+
+
+class TestEsDB(unittest.TestCase):

```

Comment:
I know this isn't a helpful comment, but maybe more positive tests wouldn't hurt. Like testing `add`, `get`, `reset`, and not just testing illegal methods.

Reply:
yes we should have both positive and negative tests

Reply:
Need help here, need to figure out how to mock elasticsearch client to successfully test both positive and negative test cases.

Reply:
@pc9 : can you open a new issue for this?

Reply:
sure will do that.